psalm.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
```

Comment:
Can we rename this file to psalm.xml.dist?

Reply:
Done!